### DISCUSSION TITLE
img tag in html not working

### DISCUSSION THREAD

**Discussion by HlaingMinThan** [spatie/browsershot#605] [Q&A]
I'm  adding my logo image in html.it's perfectly rendered in browser.but when we export with browsershot, It's broken.Is anyone has solution.i can't find any reference in docs also.

**Comment by josejbarneto** [comment]
I'm having the same problem but only in production, when i use browsershot in localhost <img> are rendering but not in production environment. The img src is absolute and the same in both environments

(exporting pdf)

![image](https://user-images.githubusercontent.com/22933565/231744008-7a4e5603-ec6e-4b79-9c31-93e8c25de6ba.png)

**Comment by hyphen81** [comment]
having the same problem, anyone else find a solution?

**Comment by martin-ro** [comment]
Yep, same here

**Comment by Quentinho** [comment]
Hi guys and everyone having this trouble, i had the same problem with html good rendered in the browser but when i generated the pdf in a production environment with this html, the images were broken.

To fix this, in my html i didn't call the images with url but with path. But this solution work only if you have the images hosted on your server, if you want to insert images from the web this solution does not work.

`
<img src="url-of-image" /> --> Before
<img src="path-on-server-to-image" /> --> After
`

I did not investigate why calling images via url does not work on my prod server (no time and not full access), but my suggestion is maybe the https is blocking the calls (because in dev i work with docker and i didn't have https activated, and i have no problem to generate the pdf). But i think there is a way to use urls instead of paths, it must be
